mock-abis/src/main/java/io/mosip/proxy/abis/controller/ProxyAbisController.java

```diff
@@ -47,6 +36,36 @@
 	@Autowired
 	ProxyAbisInsertService abisInsertService;
 
+	@RequestMapping(value = "configure", method = RequestMethod.GET)
+	@ApiOperation(value = "Configure Request")
+	public ResponseEntity<Object> checkConfiguration()
+			throws Exception {
+		logger.info("Configure Request");
+		try {
+			ConfigureDto configureDto = new ConfigureDto();
+			configureDto.setFindDuplicate(ProxyAbisInsertServiceImpl.overrideFindDuplicate);
+			return new ResponseEntity<Object>(configureDto, HttpStatus.OK);
+		} catch (RuntimeException exp) {
+			logger.error("Exception while getting configuration: "+exp.getMessage());
+			return new ResponseEntity<Object>("Failed to get configuration: "+exp.getMessage(), HttpStatus.INTERNAL_SERVER_ERROR);
+		}
+	}
+
+	@RequestMapping(value = "configure", method = RequestMethod.POST)
+	@ApiOperation(value = "Configure Request")
+	public ResponseEntity<Object> configure(@Valid @RequestBody ConfigureDto ie, BindingResult bd)
+			throws Exception {
+		logger.info("Configure Request");
+		try {
+			ProxyAbisInsertServiceImpl.overrideFindDuplicate = ie.getFindDuplicate();
```

Comment:
Create a member function to set & get "findDuplicate". Use that API to set the value.